paasta_tools/setup_istio_mesh.py

```diff
@@ -188,11 +220,12 @@ def setup_kube_services(
             )
         except Exception as err:
             log.error(f"{err} while setting up unified service")
+            traceback.print_exc()
             return False
 
     return setup_paasta_namespace_services(
         kube_client, namespaces.keys(), existing_kube_services_names, rate_limit
-    )
+    ) and cleanup_kube_svc(kube_client, namespaces.keys(), existing_kube_services_names)
```

Comment:
let's not clump this all into one line, i'd rather `return True` in the end and call setup_ and cleanup_ on separate lines

Reply:
Do you mean something like this ?
```
    setup_paasta_namespace_services_staus = setup_paasta_namespace_services(
        kube_client, namespaces.keys(), existing_kube_services_names, rate_limit
    )

    cleanup_kube_svc_status = cleanup_kube_svc(kube_client, namespaces.keys(), existing_kube_services_names)

    return setup_paasta_namespace_services_staus and cleanup_kube_svc_status
```

Reply:
something like that